RubberduckTests/Grammar/ResolverTests.cs

```diff
@@ -5121,5 +5176,85 @@ End Sub
                 Assert.AreEqual(1, defaultMemberReference.DefaultMemberRecursionDepth);
             }
         }
+
+        [Test]
+        [Category("Grammar")]
+        [Category("Resolver")]
+        [TestCase("42, a+b", "arg2", "a+b")]
+        [TestCase("42, a+b", "arg1", "42")]
+        [TestCase("arg2:=42, arg1:=a+b", "arg2", "42")]
+        [TestCase("arg2:=42, arg1:=a+b", "arg1", "a+b")]
+        [TestCase("42, a+b, (\"Hello\" & 42)", "arg3", "(\"Hello\" & 42)")]
+        [TestCase("42, a+b, (\"Hello\" & 42), 15+2", "furtherArgs", "15+2")]
+        public void CorrectArgumentReferencesOnMethodAccess(string arguments, string parameterName, string expectedExpressionText)
```

Comment:
[TestCase("42", "arg2", , "a+b")]

Reply:
I should actually also add a test for the inspection for missing arguments, and for address of expressions.